packages/broccoli-eyeglass/src/broccoli_sass_compiler.ts

```diff
@@ -627,26 +627,33 @@ export default class BroccoliSassCompiler extends BroccoliPlugin {
    * @return hash object of the file data
    **/
   hashForFile(absolutePath: string): string {
-    let cacheKey = `hashForFile(${absolutePath})`;
-    let cachedHash = this.buildCache.get(cacheKey);
-    if (cachedHash) {
-      return cachedHash as string;
+    return this.fileKey(absolutePath);
+  }
+
+  /* compute a key for a file that will change if the file has changed. */
+  fileKey(file: string, isRealPath = false): string {
+    let cachedKeyKey = `fileKey(${file})`;
+    let cachedKey = this.buildCache.get(cachedKeyKey) as string;
+    if (cachedKey) { return cachedKey; }
+    let stat = statSync(file);
+    let key;
+    if (!isRealPath && stat.isSymbolicLink) {
```

Comment:
this conditional may not be correct, I believe`stat.isSymbolicLink`  is a method, not a boolean. Are you intending to check for the existence of the the method, of if the stat itself is a link?

Reply:
That explains the infinite loop I got lol.